app/routes/labs.missions/route.tsx

```diff
@@ -0,0 +1,95 @@
+import type { BreadcrumbHandle } from '~/root/Title'
+
+import fleet from './AstroFleetChart.png'
+import meatball from '~/components/meatball/logo.svg'
+
+export const handle: BreadcrumbHandle = {
+  breadcrumb: 'Missions',
+}
+
+export default function () {
+  return (
+    <>
+      <img
+        alt="Fermi satellite with a GRB in the background, both positioned over earth"
+        src={fermiHeroImage}
+      />
+      <GridContainer>
+        <Grid row className="border-bottom border-gray-50">
+          <h1>Mutimessenger missions</h1>
+        </Grid>
+        <Grid row>
+          <h3>Overview</h3>
+          <p>
+            The joint discovery of gravitational waves and electromagnetic
+            radiation from the binary neutron star merger GW170817 was a
+            watershed moment for astrophysics. NASA missions played a critical
+            role in this discovery, from constraining the speed of gravity, to
+            determining the site of heavy (r-process) element formation, to
+            furthering our understanding of the formation and structure of
+            relativistic jets. The recent detection of a neutrino correlated in
+            space and time with a flare from gamma-ray blazar has also provided
+            a tantalizing clue to the origin of high-energy cosmic neutrinos.
+            These studies of astrophysical transients and time-domain and
+            multimessenger phenomena are perhaps the most rapidly growing field
+            of astrophysics, rich with opportunities for exciting discoveries.
+            In the present/near-future, NASA is well-positioned to capitalize on
+            the exciting scientific opportunities in time-domain and
+            multimessenger astrophysics. As highlighted in the 2019 Astrophysics
+            Senior Review of Operating Missions, the portfolio provides a suite
+            of capabilities that is “greater than the sum of its parts”, and
+            will contribute significantly to the major science questions in this
+            field.
+          </p>
+        </Grid>
+        <Grid row>
+          <div>
+            <img
+              alt="a diagram depicting all the missions in the fleet over time shaped as a swirl"
+              src={fleet}
+            />
+            <figcaption className="text-italic">
+              The current NASA Astrophysics Fleet, flying and in development.
+            </figcaption>
+          </div>
+        </Grid>
+        <Grid>
+          <div className="border-left-05 border-indigo">
+            <p className="padding-left-1">
+              "The complementary nature of these missions makes the overall
+              capability of the portfolio more than the sum of its parts, and
+              many of the most exciting developments in contemporary
+              astrophysics draw on observations from several of these
+              observatories simultaneously."
+            </p>
```

Comment:
I'm quite surprised that USWDS doesn't provide styling for `<blockquote>`. Have you asked on their Slack or submitted an issue about making a component for that?

Reply:
They don't provide a quote, a hero image, or any kind of citation. I can ask.

Reply:
The reason that I'm struggling with this PR is that the typography differs radically from USWDS design elements. I understand that @dankocevski has a vision of a graphics-intensive, outreach-focused style, but I think that you can achieve that while still using mostly stock USWDS components. We _want_ this site to mostly look like every other federal agency web site.

Why does it matter? From https://designsystem.digital.gov/about/key-benefits/#proven-design-solutions-that-users-expect-2:

> Don’t reinvent the wheel when it comes to buttons, forms, and other common elements. Provide a more seamless experience across both your agency and the entire federal government.
>
> **The design system helps users know what to expect when they visit federal websites.** It allows you to provide consistent usability, accessibility, editorial style, and look-and-feel across your websites and digital services. **This consistency delivers a more seamless, user-friendly experience** (a key requirement of [21st Century Integrated Digital Experience Act (21st Century IDEA) [digital.gov]](https://digital.gov/resources/21st-century-integrated-digital-experience-act/)) and also **increases trust in government**.

Emphasis is my own.